text/0000-javascript-module-api.md

```diff
@@ -0,0 +1,1131 @@
+| `Ember.computed.max`                  | `import { max } from "@ember/object/computed"`                             |
+| `Ember.computed.min`                  | `import { min } from "@ember/object/computed"`                             |
+| `Ember.computed.none`                 | `import { none } from "@ember/object/computed"`                            |
+| `Ember.computed.not`                  | `import { not } from "@ember/object/computed"`                             |
+| `Ember.computed.notEmpty`             | `import { notEmpty } from "@ember/object/computed"`                        |
+| `Ember.computed.oneWay`               | `import { oneWay } from "@ember/object/computed"`                          |
+| `Ember.computed.or`                   | `import { or } from "@ember/object/computed"`                              |
+| `Ember.computed.readOnly`             | `import { readOnly } from "@ember/object/computed"`                        |
+| `Ember.computed.reads`                | `import { reads } from "@ember/object/computed"`                           |
+| `Ember.computed.setDiff`              | `import { setDiff } from "@ember/object/computed"`                         |
+| `Ember.computed.sort`                 | `import { sort } from "@ember/object/computed"`                            |
+| `Ember.computed.sum`                  | `import { sum } from "@ember/object/computed"`                             |
+| `Ember.computed.union`                | `import { union } from "@ember/object/computed"`                           |
+| `Ember.computed.uniq`                 | `import { uniq } from "@ember/object/computed"`                            |
+| `Ember.copy`                          | `import { copy } from "@ember/object/internals"`                           |
+| `Ember.create`                        | `import { create } from "@ember/polyfills"`                                |
+| `Ember.debug`                         | `import { debug } from "@ember/debug"`                                     |
+| `Ember.deprecate`                     | `import { deprecate } from "@ember/application/deprecations"`              |
+| `Ember.deprecateFunc`                 | `import { deprecateFunc } from "@ember/application/deprecations"`          |
+| `Ember.get`                           | `import { get } from "@ember/object"`                                      |
+| `Ember.getOwner`                      | `import { getOwner } from "@ember/application"`                            |
+| `Ember.getProperties`                 | `import { getProperties } from "@ember/object"`                            |
+| `Ember.guidFor`                       | `import { guidFor } from "@ember/object/internals"`                        |
+| `Ember.inject.controller`             | `import { inject } from "@ember/controller"`                               |
+| `Ember.inject.service`                | `import { inject } from "@ember/service"`                                  |
```

Comment:
I noticed that the migration utility uses the `as` trick to retain this type of syntax:

```js
// old
import Ember from ember;
{
  foo : Ember.inject.service()
}

// new 
import { inject as service } from "@ember/service";

{
  foo: service()
}
```

If injecting a controller and a service in the same module, people need to learn about named imports using the `as` keyword. 

I also somewhat like seeing services injected at the top of an Ember component vs. just the word "inject".

```js
// old
{
  store: service(),
  session: service(),
  location: service(),
}

// proposed new
{
    store: inject(),
    session: inject(),
    location: inject(),
}
```

IMO it makes it easier to read & know "this is the session service and I can find the definition in `app/services/session.js`.